cmd/queue/main.go

```diff
@@ -311,9 +311,14 @@ func buildServer(ctx context.Context, env config, drainer *pkghandler.Drainer, p
 	if tracingEnabled {
 		composedHandler = tracing.HTTPSpanMiddleware(composedHandler)
 	}
+	if env.ServingEnableRequestLog {
+		composedHandler = requestLogHandler(logger, composedHandler, env)
```

Comment:
I'd be tempted to move this down in to the other if env.ServingEnableRequestLog block since it's only 3 lines down from here. I'm OK to leave it here if you disagree, though.

Reply:
my point was to keep each handler in its own block of code

Reply:
👍 fine with me, I don't feel super stronly